### DISCUSSION TITLE
Good microphones to use in okada?

### DISCUSSION THREAD

**Discussion by gworkman27** [w-okada/voice-changer#1270] [General]
So far, I have been just using my built in computer microphone to record my own voice before changing it in okada. Has anyone found that they got better quality recordings when they used a microphone? What microphone do you recommend?

**Comment by AnnoyingFox** [comment]
The way RVC works, the quality of the mic doesn't matter. You won't get better results with a better mic. In fact, having a "too good" mic will only result in RVC picking every little noise that you do which will then be translated into little weird noises in RVC because it will try to convert every sound into "voice".

So yeah, whatever headset mic will work. As long as what you speak is clear enough to understand, RVC will understand it too. Keep in mind that RVC **REPLACES** what you say so the quality of the mic haven't any impact on the quality of the output (unless the mic is so bad that the sound is broken lol)

**Reply by gworkman27** [reply]
Very good to know. A better mic will not improve audio quality.

**Reply by AnnoyingFox** [reply]
> Very good to know. A better mic will not improve audio quality.

My friend does have one of those high quality streamers mic. When she uses it with RVC, since the quality of the mic pics every little noise and detail, something as simple as moving in her chair or even typing with the keyboard is registered by RVC which tries to convert those noises into phonemes. So you end hearing a lot of randoms "ah, ugh, eh, gah" lol. You can avoid this using noise filters either from the mic software or the ones included in RVC but I use the mic included in my Logitech Pro X wireless headset and this is unlikely to happen so often because it doesn't pick so much of the noise around. So yeah, is not worth to invest on a high quality mic unless you also want to use it without RVC.

**Reply by Exitiumlion** [reply]
This is old thread, but not entirely true what I'm reading here. A better mice is best with noise cancelation and noise gate with filters. A headset that's good also has cancelation built-in or using software. A bad mic with noise will not convert your voice clearly and might add "ah, ugh, eh, gah" w ith

**Reply by AnnoyingFox** [reply]
More on the contrary, the better the mic, the more noises it will pick. It does not mean that you have to get the worst mic to sound the best, but you don't need a professional expensive mic. I know because I have tried both and any headset one does a better job. Noise cancelation filters are all software side, and as such, it has nothing to do with the quality of the mic. RVC will convert your voice and that's why it doesn't matter much the quality input as long as it's good enough for a clear speaking. You will get the same result using a 50$ mic than using a 5000$ one. RVC already includes noise cancelation built-in and it's good enough. You can also easily use nvidia broadcast for that purpose. But if you want to think otherwise go ahead. Just don't deceive users saying that they will get better quality conversion having a better mic.

**Reply by Exitiumlion** [reply]
My response to  "The way RVC works, the quality of the mic doesn't matter." is that it's not entirely true. Garbage in = Garbage out.
A better mic is better, never mentioned price or professionalism brand. A built in computer microphone can't beat alternatives. 

Continuing, mic noise and quality of voice being sent to RVC, not background sounds that get picked up by mic. Bad cheap mic's are terrible and have "hum" or static noise that gets sent to the RVC. Instead of using built-in noise cancelation heavily that could also effect voice. Why not use a better mic **with** lighter or no noise cancelation to preserve as much of your voice for conversion.

NOW if you are in a background noisy environment (Fans, cars) then getting a better mic to deal with it will disappoint you. As it will do it's job and pick up sound. Unless you can eliminate these backgrounds then Noise cancelation is your only friend in these environments. Yet a better mic would still be better than built in laptop mic. like look for a headset to use, as few have good mic with hardware cancelation that would cut some of it (hopefully digital signal processing vs two mics spliced) also having the mic closer to mouth could help cut the voice out of the noise and make it clear to RVC after software noise cancelation. Play with the raw input levels to find a balance.

My set up is with a mic at 69% level input to cut noise vs signal. Sent to RVC raw with no noise cancelation but only with cut off. Works well and clean. do i get "ah, ugh, eh, gah"...  quietly. As the mic does pick up noise but rarely as the mic can't pick it up with 69% level input. Even when RVC picks it up that's fine and doesn't bother people. It's my setup, and I can easily turn on noise cancelation. My older mic.... caused so much problems that i don't think most don't understand how bad it can get with poor mic, where you can't separate the noise and voice let alone background noise worries. 
My recommendation, look at reviews for mic's or headsets and get the cheapest that don't sound bad or noisy. 

Also $50 bucks is already in the ballpark of good mic and many are in that range. You can get a $25 Lavalier Microphone that rocks, they work super well and oddly good mic's for the price. A $60 headset like the "HyperX Cloud III" comes with a good mic for VC or RVC.

**Comment by AnnoyingFox** [comment]
I am sure I already mentioned that it haven't to be a bad mic. You are just saying what I have already explained. A 50$ mic is definitely an average mic, not a good mic. It doesn't means that it's bad, it's good for it's price. But if you think that that's a good mic then I have bad news for you. A good mic will more likely be past 100 and even 200 easily. So, like I said for like three times, a better mic won't ever improve RVC qualify as long as the input is clear. A good mic will always pic way more noise than any other. It just make use of noise cancelation through software. There is a reason why good mics are used in a soundproof environment with pop filters. They can capture more and better details but only in the right environment. For RVC none of that it's necessary because everything will be converted. And it's already pretty hard to get a bad mic nowadays unless you use no mic and rely on a laptop or Webcam embedded one. But I never mentioned that these are good for RVC. Your issue is that you aren't understanding the difference between a good and an "OK" mic. I have several of them, I have a blue yeti expensive one, I have the included in the headset from the new Artic Nova Pro, and I also have the one from the logitech X pro. And the one that gives worse results by default is the yeti one which is of the highest quality. Having to filter noise and everything that can bug RVC will end with the same quality of an average mic so isn't worth the extra money. It's that simple.

If you are going to use RVC in a studio like environment with sound proof walls and pop filters then go ahead and invest in a good mic. Otherwise the quality of any headset, even a cheap one, will do the job perfectly fine. So when even a 20$ headset gives you good results, there is no reason to day that better mics will work better with RVC. Nowadays there aren't even "bad" mics unless you are trying to use, like mentioned above, Webcam and laptop embedded ones which aren't either bad, just not placed in the right spot.

I won't keep debating about this subject because it's quite unrelated to RVC in many ways. Just use whatever mic you want.
